packages/paste-website/src/pages/components/button/index.mdx

```diff
@@ -229,17 +231,19 @@ their default size.
 
 <LivePreview scope={{Button, Box}} language="jsx">
   {`<div>
-  <Box display="inline-block" mr="space30">
+  <Box display="inline-block" marginRight="space30">
     <Button variant="primary" size="small">
       Submit
     </Button>
   </Box>
 
-  <Box display="inline-block" mr="space30">
-    <Button variant="secondary" size="small">Submit</Button>
-  </Box>
+<Box display="inline-block" marginRight="space30">
```

Comment:
What are you using to auto-format these MDX files? This is going to keep happening if we aren't all using the same thing

Reply:
One (or both?) of these VS Code addons 
![image](https://user-images.githubusercontent.com/1592327/66506794-9e64a980-ea93-11e9-8df1-5bf363f2fdb5.png)

Reply:
Hmm, I have both of those, and if it was prettier I'd expect the build to fail as we run prettier on our files in circle so it can't be that either... mystery 🤔

Reply:
Mine doesn't do any auto-formatting at all?! I'm missing out on so much!

Reply:
Have you both got some kind of local prettier config going on, as our config ignores mdx files entirely https://github.com/twilio-labs/paste/blob/master/.prettierignore#L2

Reply:
In my settings (cmd + ,) I have
![image](https://user-images.githubusercontent.com/1592327/66519282-a67d1300-eaac-11e9-8182-3a2ec2577d27.png)

![image](https://user-images.githubusercontent.com/1592327/66519295-ada42100-eaac-11e9-84bc-d77cc24baf49.png)

and 
![image](https://user-images.githubusercontent.com/1592327/66519331-b85eb600-eaac-11e9-8f1a-dc3376fd4677.png)

Reply:
Does VS Code auto-format a JSON file for you?

Reply:
I'm running the exact same settings as @TheSisb.

Reply:
Looks like VS Code is formatting JSON on my end.

Reply:
It very much shouldn't be. Your VS Code is ignoring our `.prettierignore` file

Reply:
I think the `.prettierignore` file is just for CLI prettier (so our yarn scripts and CI) but user level configs override those.

Reply:
Its seems like thats whats happening.

Reply:
I'm going to un ignore those files in another branch